charts/geth/templates/_cmd.tpl

```diff
@@ -27,9 +27,14 @@
   --ws.addr=0.0.0.0
   --ws.port={{ include "geth.wsPort" . }}
   --ws.origins=*
+  --authrpc.jwtsecret=/data/jwt.hex
+  --authrpc.addr=0.0.0.0
+  --authrpc.port={{ include "geth.authPort" . }}
+  --authrpc.vhosts=*
   --metrics
   --metrics.addr=0.0.0.0
   --metrics.port={{ include "geth.metricsPort" . }}
+  --syncmode snap
```

Comment:
Use `extraArgs` withing values.yaml for a specific deployment flags instead of adding this here as default.  Since "snap" is the actual default, you actually don't need it at all.